src/javascripts/views/homeView.js

```diff
@@ -0,0 +1,15 @@
+class HomeView {
+    constructor() {
+        this.transactionListEl = document.querySelector(".transaction-list")
+        this.addBtnEL = document.querySelector(".btn-add-transaction");
```

Comment:
El not EL

Reply:
I fixed @phuocduong-agilityio . Please help me check. Thanks!